### ISSUE TITLE
FNX-14606 ⁃ [webcontent] Twitter broken by Tracking Protection

### ISSUE THREAD

**Issue by vishalnandagopal** [mozilla-mobile/fenix#13425]
## Site with issue and any steps to reproduce

Site - mobile.twitter.com 
Image - 

https://i.postimg.cc/wMpx5hJG/Imagepipe-0.jpg

### Expected behavior
Tracking Protection should not block abs.twimg.com on Twitter.

### Actual behavior

FF blocks abs.twimg.con

### Does toggling Tracking Protection fix the issue? (Press the lock icon in the toolbar while on the site to see toggle)

Yes

### Can you reproduce in [Firefox for Android](https://play.google.com/store/apps/details?id=org.mozilla.firefox)?

Yes(Firefox Beta) . This problem also occurred on desktop FF. So definitely a tracking protection problem. 

### Can you reproduce in Chrome (or other non-Mozilla browser)?
No.

### Device information

* Android device: 10
* Fenix version: Nightly 200807 06:01 (Build #2015756531)

**Comment by kbrosnan** [comment]
Requires bz-1446243

**Comment by LinAGKar** [comment]
I ran into a similar problem now. Firefox inexplicably refused to load mobile.twitter.com while tracking protection was set to strict, instead giving an error message saying:

```
Begäran kan inte slutföras

Ytterligare information om detta problem eller fel är för närvarande inte tillgänglig.
```

And showing the connection as insecure.

It worked after setting tracking protection to standard, and kept working after setting it back to strict.

**Comment by Powersource** [comment]
Yep getting the same as the above comment

**Comment by vishalnandagopal** [comment]
>not able to reproduce this issue on the latest Firefox Nightly build from 3/31

This bug happened happened almost a year ago. It does not happen now, so cannot be reproduced. I assume it was fixed during these months.

I had attached a screenshot to the my issue showing what was blocked by Tracking Protection.

![image](https://user-images.githubusercontent.com/61023404/114317523-1ea31980-9b26-11eb-9624-5ae5c19d472f.png)

I should have closed this issue when Twitter started working on Firefox Mobile again. Apologize for the delay in closing the issue and replying to your comments, I don't use GitHub a lot.
